.gitignore

```diff
@@ -1,6 +1,9 @@
 # Temporary Build Files
-build/_output
-build/_test
+build/
```

Comment:
There are few files under the `build` which need to be tracked, right?

Reply:
We have Dockerfile inside build. I think this file need to be tracked.

Reply:
We need that Dockerfile - yes!